src/core_plugins/kbn_vislib_vis_types/public/controls/gauge_options.js

```diff
@@ -0,0 +1,110 @@
+import { uiModules } from 'ui/modules';
+import gaugeOptionsTemplate from 'plugins/kbn_vislib_vis_types/controls/gauge_options.html';
+import _ from 'lodash';
+const module = uiModules.get('kibana');
+
+module.directive('gaugeOptions', function () {
+  return {
+    restrict: 'E',
+    template: gaugeOptionsTemplate,
+    replace: true,
+    link: function ($scope) {
+
+      $scope.showColorRange = true;
+
+      $scope.$watch('vis.params.gauge.gaugeType', type => {
+        switch (type) {
+          case 'Meter':
+            $scope.vis.params.gauge.type = 'meter';
+            $scope.vis.params.gauge.minAngle = undefined;
+            $scope.vis.params.gauge.maxAngle = undefined;
+            break;
+          case 'Circle':
+            $scope.vis.params.gauge.type = 'meter';
+            $scope.vis.params.gauge.minAngle = 0;
+            $scope.vis.params.gauge.maxAngle = 2 * Math.PI;
+            break;
+          case 'Metric':
```

Comment:
there's a small bug when switching to metric:

```
Error: Accessing invalid config property: fontSize
    at VisConfig.get (https://localhost:5601/atr/bundles/kibana.bundle.js?v=8467:112087:18)
    at SimpleGauge.drawGauge (https://localhost:5601/atr/bundles/kibana.bundle.js?v=8467:322559:59)
    at https://localhost:5601/atr/bundles/kibana.bundle.js?v=8467:322042:40
    at Array.forEach (native)
    at HTMLDivElement.<anonymous> (https://localhost:5601/atr/bundles/kibana.bundle.js?v=8467:322037:26)
    at https://localhost:5601/atr/bundles/kibana.bundle.js?v=8467:96916:17
    at d3_selection_each (https://localhost:5601/atr/bundles/kibana.bundle.js?v=8467:96922:31)
    at Array.d3_selectionPrototype.each (https://localhost:5601/atr/bundles/kibana.bundle.js?v=8467:96915:13)
    at Array.<anonymous> (https://localhost:5601/atr/bundles/kibana.bundle.js?v=8467:322030:22)
    at Array.d3_selectionPrototype.call (https://localhost:5601/atr/bundles/kibana.bundle.js?v=8467:96929:15)
```
To reproduce:
1. choose a gauge-viz
2. switch to metric gauge-type, press play
3. -> screen goes blank